kratos/utilities/dof_updater.h

```diff
@@ -0,0 +1,162 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Jordi Cotela
+//
+
+#if !defined(KRATOS_DOF_UPDATER_H_INCLUDED )
+#define  KRATOS_DOF_UPDATER_H_INCLUDED
+
+
+// System includes
+#include <string>
+#include <iostream>
+
+
+// External includes
+
+
+// Project includes
+#include "includes/define.h"
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+///@addtogroup KratosCore
+///@{
+
+///@name Kratos Classes
+///@{
+
+/// Short class definition.
+/** Detail class definition.
+*/
+template< class TSparseSpace >
+class DofUpdater
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    /// Pointer definition of DofUpdater
+    KRATOS_CLASS_POINTER_DEFINITION(DofUpdater);
+
+    using DofsArrayType = ModelPart::DofsArrayType;
+    using SystemVectorType = typename TSparseSpace::VectorType;
+
+    using UniquePointer = std::unique_ptr<DofUpdater>;
```

Comment:
Does a `Kratos::unique_ptr` exist?
@roigcarlo

Reply:
it does

Reply:
@RiccardoRossi I could not find it... Do you know where it is?